api/api.go

```diff
@@ -693,10 +704,13 @@ func procAttachBackend(c libnetwork.NetworkController, vars map[string]string, b
 		setFctList = append(setFctList, libnetwork.CreateOptionAlias(name, alias))
 	}
 
+	log.Printf("Entering api join ...")
```

Comment:
I am guessing these logs were added for debugging. If so you may want to remove them.